### PR TITLE
feat: only cog historical imagery

### PR DESCRIPTION
**Pull request by Ben-Cheng**

### CODE REVIEW

File: topo_processor/cli/main.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import click
 from linz_logger import get_log
 
-from topo_processor.metadata import DataType, create_collection
+from topo_processor.factory import DataType, create_collection
```
Comment (blacha) [inline topo_processor/cli/main.py:9]: factory is super generic name when its creating metadata why the change?

File: topo_processor/imagery/compressors/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from .data_compressor_imagery_historic import DataCompressorImageryHistoric
```
Comment (blacha) [inline topo_processor/imagery/compressors/__init__.py:1]: might be better to call this a `transformer` and the have a compressor as type of a transformer

File: topo_processor/cli/main.py (hunk 1 above)
Comment (Ben-Cheng) [reply topo_processor/cli/main.py:9]: because it also calls compress data / tiffs in create_collection

File: topo_processor/imagery/compressors/__init__.py (hunk 2 above)
Comment (Ben-Cheng) [reply topo_processor/imagery/compressors/__init__.py:1]: why is that? are we going to have other types of transformer in the future for lidar and aerial imagery?

Comment (blacha) [reply topo_processor/imagery/compressors/__init__.py:1]: Yes there will be more transformations later, cropping/filtering also a gdal_translate is a transformation, you don't actually have to compress a file to make a COG.

Comment (Ben-Cheng) [reply topo_processor/imagery/compressors/__init__.py:1]: yup will change that

File: topo_processor/cli/main.py (hunk 1 above)
Comment (Ben-Cheng) [reply topo_processor/cli/main.py]: I have renamed it to stac_object because they are mainly for creating collections and items. Thoughts? @blacha

Comment (Ben-Cheng) [reply topo_processor/cli/main.py]: have renamed it to stac

**Review by blacha — APPROVED** [review]
